djangocms_admin_style/sass/components/_sideframe.scss

```diff
@@ -1,4 +1,7 @@
 .cms-admin-sideframe {
+    #header {
+        display: none;
```

Comment:
@vxsx yes because then the header is not doubled. The links are already in the toolbar